adoc/Release-Notes.adoc

```diff
@@ -109,81 +109,52 @@ For information about deploying and administering {product}, see the product man
 ====
 In order to deploy {product} 1.5 or upgrade from {product} 1.4.1 in an HA configuration, you will first need to scale the `mysql` role down to 1 instance to be able to migrate from `cf-mysql` to `PXC`. This is based on upstream instructions but based on what we've seen with other components that rely on the database, such as Credhub, scaling all database roles into single availability helps with a stable migration and deployment.
```

Comment:
Sorry, one other place where the wording needs to be fixed - "In order to deploy {product} 1.5 or upgrade from {product} 1.4.1 in an HA configuration, you will first need to ~~scale the `mysql` role down to 1 instance~~ to be able to migrate from `cf-mysql` to `PXC`." Maybe change this to a more accurate wording of "...first need to **start the mysql role with 1 instance**..."

Reply:
Fixed in https://github.com/SUSE/cap-release-notes/pull/84/commits/5fae74ac5fff0c0455731356b5ae3459a764d44c